packages/orderbook/test/js_api/common.test.ts

```
@@ -0,0 +1,170 @@
+		assert.equal(
+			res.error.msg,
+			'deployers.some-deployer: missing field `address` at line 3 column 17'
+		);
+		assert.equal(
+			res.error.readableMsg,
+			'Failed to parse yaml configuration: deployers.some-deployer: missing field `address` at line 3 column 17'
+		);
+	});
+
+	it('should get remove order calldata', async () => {
+		let order: SgOrder = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'order' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)